lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1553,7 +1553,7 @@ def cmd_set(*args)
     # If the driver indicates that the value is not valid, bust out.
     if (driver.on_variable_set(global, name, value) == false)
       print_error("The value specified for #{name} is not valid.")
```

Comment:
Nothing does, actually! The dispatcher shell doesn't check the retval of any of these methods. I assume a retval is provided in case someone actually wants to use it. I'm inverting the value because error conditions from command dispatcher methods should return `false`. I assume this was a typo.

Reply:
https://github.com/rapid7/metasploit-framework/blob/d6bce4410c27c7d8e25bf8f8bd575d5dfe55dae5/lib/rex/ui/text/dispatcher_shell.rb#L537-L552

Reply:
OK, ounds reasonable to change it then. Thanks!